app/controllers/api/v1/providers_controller.rb

```diff
@@ -4,37 +4,47 @@ class ProvidersController < ApplicationController
       include NextLinkHeader
 
       # Potential edge case:
-      # It is possible for older updated_at values to written to the database after this API has been queried for changes. This would mean that these changes are missed when the client makes a subsequent request using the next-link.
+      #
+      # It is possible for older updated_at values to written to the database
+      # after this API has been queried for changes. This would mean that these
+      # changes are missed when the client makes a subsequent request using the
+      # next-link.
+      #
       # Possible causes of older updated_at values:
-      # - delay between c# calculating datetime.UtcNow and value being written to postgres
+      # - delay between c# calculating datetime.UtcNow and value being written
+      #   to postgres
       # - clock drift between servers
       def index
         per_page = params[:per_page] || 100
         changed_since = params[:changed_since]
-
         ActiveRecord::Base.transaction do
           ActiveRecord::Base.connection.execute('LOCK provider, provider_enrichment, site IN SHARE UPDATE EXCLUSIVE MODE')
-          @providers = Provider.opted_in.changed_since(changed_since).limit(per_page)
+          @providers = Provider
+                         .opted_in
+                         .changed_since(changed_since)
+                         .limit(per_page)
         end
 
         last_provider = @providers.last
 
         response.headers['Link'] = if last_provider
-                                     next_link((last_provider.last_published_at + 1.second).utc.iso8601, last_provider.id, per_page)
+                                     next_link(last_provider.changed_at
+                                                 .utc
+                                                 .strftime('%FT%T.%6NZ'),
```

Comment:
I'd like a comment explaining what this format is and why to save going to look it up, maybe with a url for the docs